.gitignore

```diff
@@ -109,6 +109,9 @@ resource_validation_results.json
 
 # Server files
 license/
-vonk-trial-license.json
 appsettings.env
```

Comment:
Can you also remove `license`, `appsettings.env`, and `logsettings.env` (basically everything under the comment Server files) Thanks!